_layouts/default.html

```diff
@@ -39,6 +39,9 @@
     <script src="{{ site.baseurl }}/assets/js/main.js"></script>
     <script src="{{ site.baseurl }}/assets/js/gaEvents.js"></script>
 
+    {% assign ga_id = site.google_analytics[site.branch]|default:site.google_analytics.default %}
+    {% if ga_id %}{% include google_analytics.html id=ga_id %}{% endif %}
```

Comment:
If you only have one thing to keep track of for GA, you could replace line 42 and 43 with :
```{% include google_analytics.html %}```
and you wouldn't have to add to the `_config.yml`